lib/src/models/rover/controller.dart

```diff
@@ -23,15 +23,29 @@ class Controller extends Model {
 	RoverControls controls;
 
 	/// Whether the start button has been pressed.
-	/// 
+	///
 	/// When the start button is released, the dashboard will switch to the next mode.
+
 	bool isStartPressed = false;
 
+	/// Map to figure out what device is connected
+	/// <Teensy, Command>
+	/// Used to send data to the correct teensy
+	Map<String, String> teensyCommands = {
```

Comment:
Try to follow our convention of: 
```dart
/// One sentence briefly describing what this is
/// 
/// More sentences after a blank line
```
(Also, it's `Command => Teensy` now)

Reply:
Also, you don't want the Teensy's to be strings, because we don't use those strings anywhere else. There's nothing to compare them to. We use the `Device` enum, so this should be a `Map<String, Device>`

Reply:
the models.serial.connectedDevice is a string tho,
and models.serial.availablePorts is string iterable.

Reply:
I didn't realize we had `models.serial.connectedDevice`. But in any case, you want to use `services.serial.connectedDevice`. See my new review below.